sympy/calculus/finite_diff.py

```diff
@@ -313,25 +312,23 @@ def _as_finite_diff(derivative, points=1, x0=None, wrt=None):
     Examples
     ========
 
-    >>> from sympy import symbols, Function, exp, sqrt, Symbol, as_finite_diff
-    >>> from sympy.utilities.exceptions import SymPyDeprecationWarning
-    >>> import warnings
-    >>> warnings.simplefilter("ignore", SymPyDeprecationWarning)
+    >>> from sympy import symbols, Function, exp, sqrt, Symbol
+    >>> from sympy.calculus.finite_diff import _as_finite_diff
```

Comment:
can we just use `finite_diff`? why the underscore?

Reply:
Because it earlier said `as_finite_diff` and that called `_as_finite_diff`?

I didn't look more into it, but can do.

Reply:
Not sure if I fully understand the comment. The name `as_finite_diff` was deprecated and `Derivative.as_finite_difference` should be used instead. Hence, reintroducing `finite_diff` as a public function didn't seem to be the intention then.

Maybe one should not have those code examples in the documentation for `_as_finite_diff`? Not sure if they will show up in the documentation (maybe private functions/methods shouldn't?) though, but for development documentation I think it can make sense.

Reply:
Both public and private functions are included in the docs site. 

I'm not entirely clear what this function is, but the doctests should use the same name of the function, so if the function is called _as_finite_diff that's what the doctest should use. 

This actually should have been updated to this as soon as the function was deprecated. We should always document non-deprecated behavior. Using `warnings.simplefilter("ignore", SymPyDeprecationWarning)` in a doctest is a bad sign.